net/masscan/Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2008-2017 OpenWrt.org
```

Comment:
Don't assign copyright to OpenWrt unless you're working under contract or affiliated.